tests/v2/validation/provisioning/permutations/permutations.go

```diff
@@ -151,3 +189,50 @@ func GetClusterProvider(clusterType string, nodeProviderName string, provisionin
 	}
 	return &nodeProvider, &rke1NodeProvider, &customProvider, kubeVersions
 }
+
+func createAWSWorkloadAndServices(t *testing.T, client *rancher.Client, cluster *steveV1.SteveAPIObject) *steveV1.SteveAPIObject {
+
+	status := &provv1.ClusterStatus{}
+	err := steveV1.ConvertToK8sType(cluster.Status, status)
+	require.NoError(t, err)
+
+	adminClient, err := rancher.NewClient(client.RancherConfig.AdminToken, client.Session)
+	require.NoError(t, err)
+
+	steveclient, err := adminClient.Steve.ProxyDownstream(status.ClusterName)
+	require.NoError(t, err)
+
+	nginxWorkload, err := createNginxDeployment(steveclient, status.ClusterName)
+	require.NoError(t, err)
+
+	nginxSpec := &appv1.DeploymentSpec{}
+	err = steveV1.ConvertToK8sType(nginxWorkload.Spec, nginxSpec)
+	require.NoError(t, err)
+
+	clusterIPserviceName := namegenerator.AppendRandomString("cip")
+	clusterIPserviceTemplate := services.NewServiceTemplate(clusterIPserviceName, "default", corev1.ServiceTypeClusterIP, []corev1.ServicePort{{Name: "port", Port: 80}}, nginxSpec.Selector.MatchLabels)
```

Comment:
nit; can you put `default` and `port` into a const block so we're not hardcoding?